index.html

```diff
@@ -431,6 +431,18 @@ <h4>Create (Register)</h4>
                         However, an introducer is prevented from seeing the secure and private connection built by the
                         two parties.</li>
                 </ul>
+            <p>Additional security against man-in-the-middle attacks may be achieved by signing of 
+                the public invitation.</p>
+                <ul>
+                    <li>If the public invitation includes a publicly-resolvable DID, then the invitee can verify 
+                        that the signature on the invitation matches with an active verification key for that 
+                        DID.</li>
+                    <li>If the public invitation includes a DID Document, then the invitee can verify that the signature
```

Comment:
The connection protocol does not allow a public invitation to include a DID Document, so I think this paragraph is unnecessary--unless you'd like to propose a corresponding change to the connection protocol.

We chose this deliberately, because if the invitation is observable, the DID and/or DID Doc it contains is disclosed to any observer, which undermines subsequent privacy. We do allow a DID--but the public DID must be replaced with a pairwise one in the connection-response message, so even the DID is not the same by the time the connection is built.

Reply:
That makes sense. I removed the reference to the DID Document. Better now?